### PR TITLE
Add ness alarm control panel

### PR DESCRIPTION
**Pull request by nickw444**
**Description:**

Add documentation to ness alarm component.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#18463

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Review by balloob — APPROVED** [review]

File: source/_components/alarm_control_panel.ness_alarm.markdown [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+title: "Ness Alarm Control Panel"
+description: "Instructions on how to setup the Ness alarm control panel within Home Assistant."
+date: 2018-11-25 18:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: ness.jpg
+ha_category: Alarm
+ha_release: 0.84
```
Comment (MartinHjelmare) [inline source/_components/alarm_control_panel.ness_alarm.markdown:12]: 0.85

File: source/_components/binary_sensor.ness_alarm.markdown [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+title: "Ness Alarm Binary Sensor"
+description: "Instructions on how to setup the Ness alarm zones within Home Assistant."
+date: 2018-11-25 18:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: ness.jpg
+ha_category: Binary Sensor
+ha_release: 0.84
```
Comment (MartinHjelmare) [inline source/_components/binary_sensor.ness_alarm.markdown:12]: 0.85

File: source/_components/ness_alarm.markdown [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+title: "Ness Alarm"
+description: "Instructions on how to integrate a Ness D8x/D16x alarm system with Home Assistant."
+date: 2018-11-25 18:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: ness.jpg
+ha_category: Alarm
+ha_release: 0.84
```
Comment (MartinHjelmare) [inline source/_components/ness_alarm.markdown:12]: 0.85

File: source/_components/ness_alarm.markdown [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+  required: true
+  type: string
+port:
+  description: The port on which the IP232 module listens for clients.
+  required: true
+  type: integer
+zones:
+  description: List of zones to add 
+  required: false
+  type: [integer, list]
```
Comment (MartinHjelmare) [inline source/_components/ness_alarm.markdown:60]: Isn't it just a list?

File: source/_components/alarm_control_panel.ness_alarm.markdown [hunk 5]
```diff
@@ -0,0 +1,19 @@
+---
+layout: page
+title: "Ness Alarm Control Panel"
+description: "Instructions on how to setup the Ness alarm control panel within Home Assistant."
+date: 2018-11-25 18:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: ness.jpg
```
Comment (MartinHjelmare) [inline source/_components/alarm_control_panel.ness_alarm.markdown:10]: The logo is a png file.

**Review by MartinHjelmare — COMMENTED** [review]
Please make a new PR with the adjustments targeting current.

**Pull request merged**